src/ripple/shamap/impl/SHAMapTreeNode.cpp

```diff
@@ -43,7 +43,8 @@ SHAMapInnerNode::clone(std::uint32_t seq) const
     p->mHash = mHash;
     p->mIsBranch = mIsBranch;
     p->mFullBelowGen = mFullBelowGen;
-    std::memcpy(p->mHashes, mHashes, sizeof(mHashes));
+    for (int i = 0; i < sizeof(mHashes) / sizeof(mHashes[0]); ++i)
```

Comment:
Food for thought: if `mHashes` was an `std::array` then we could do `p->mHashes = mHashes;`